pkg/operator/k8sutil/k8sutil_test.go

```diff
@@ -38,11 +38,11 @@ func TestTruncateNodeName(t *testing.T) {
 			"rook-ceph-osd-prepare-%s",                  // 22 chars (without format)
 			"k8s-worker-500.this.is.a.not.so.long.name", // 41 chars
 		},
-		"47d1eb58e063be11c11c995b74cc5fbe": { // 32 chars
+		"8a49e4ac7d82f94a97234829dbf768fe1a2e9f2453b52d3d7c7c41ecfa65e01d": { // 32 chars
```

Comment:
You also forgot to update the string length in the comment here.

```suggestion
		"8a49e4ac7d82f94a97234829dbf768fe1a2e9f2453b52d3d7c7c41ecfa65e01d": { // 64 chars
```